Creating pull request per Lesson 1 Assignment

HMalloy opened the pull request:
Probably did a lot more CSS work than was expected for this first lesson, I just can't help myself sometimes.

HMalloy committed:
created initial 4 pages

HMalloy committed:
added shared structure / content to all pages

HMalloy committed:
started styles.css and added all content to pages per mockups

HMalloy committed:
updates to HTML and added CSS

HMalloy committed:
added checkout links in appropriate places, added media queries for screens

HMalloy committed:
updates per validator feedback

MadEste commented on styles.css:
```diff
@@ -0,0 +1,428 @@
+body {
+  margin:0;
+  font-family: sans-serif;
+  font-size: 18px;
```
As you will learn later on on the course we are going to be getting away from using px since they are not responsive. Consider using % or em/rem instead.

HMalloy committed:
added fonts, updates to styles, more details, more mobile tweaks

HMalloy committed:
updated structure to account for errors when using sections, updated css to remove 'px' and use 'rem' instead

HMalloy commented:
New for Lesson 2 assignment 3:
Updated HTML to use DIVs instead of Sections where appropriate
Went rem vs px crazy in the CSS
Added google fonts
Other refinements and details

HMalloy committed:
updates to footer and header

HMalloy committed:
additional updates to header background image

HMalloy committed:
created js file, updated HTML for click events, added click event to alert dog info

HMalloy committed:
added js to make it work

HMalloy committed:
found a spelling error

HMalloy committed:
made sure alert included for adoption fees had more details

MadEste commented on script.js:
```diff
@@ -0,0 +1,17 @@
+// #cart-total
+
+function dogInfo(name,breed,cost) {
+  alert('This is ' + name + ', they are a ' + breed + '. It is $' + cost + ' to bring them home with you!');
```
Nice work! Consider using JS template literals for string concatenation. I think the syntax is nicer: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

HMalloy replied:
This is great! Thanks!

MadEste commented on script.js:
```diff
@@ -0,0 +1,17 @@
+  alert('Total: $' + total);
+
+  // a little something extra = updates cart total in header, but only for the page you are on (for now)
+  document.getElementById("cart-total").innerHTML = total;
```
👍 Great work! this is exactly what you will be doing in future assignments.

HMalloy committed:
updates to design based on new comp, updates to tablet and mobile views

HMalloy committed:
missing div close caught by HTML validator!

HMalloy committed:
some work on print media query

HMalloy committed:
fixed media queries after running through css validator

HMalloy committed:
updated background image for header, added gradient, double checked images looked good at varying sizes

HMalloy committed:
playing with the blog script

HMalloy committed:
more javascript for the blog, playing

HMalloy committed:
got the title working on the blog page

HMalloy committed:
updates to blog script and css for display on home page vs blog page

HMalloy committed:
updates to checkout - alert and console.log working on submit

HMalloy committed:
having trouble getting the functions to work when they are all in the same file

HMalloy committed:
I needed to specify to only loop through the blog posts if the blog posts could be found on the page. That fixed everything

HMalloy committed:
updating so that the home page loads less dogs than the dogs page. But don't think I did this the 'right' way

HMalloy committed:
was able to figure out a better way to list less dogs on the home page. yay.

MadEste commented on script.js:
```diff
@@ -0,0 +1,210 @@
+  }
+];
+
+if (blog) {
```
👍 Nice job checking if there is content to load.

MadEste commented on checkout.html:
```diff
@@ -0,0 +1,126 @@
+      const finish = document.getElementById('submitCheckout');
+
+      finish.addEventListener('submit', function(event){
+        event.preventDefault();
```
👍

HMalloy committed:
added new jQuery file and updated per assignment

HMalloy committed:
removed unneeded code

HMalloy committed:
added persona file, added README.md, updates per testing results

HMalloy committed:
additional updates to testing info in README.md, and updates to style.css per testing results

HMalloy committed:
updated personas, so there was more than 1, updates to readme file after testing on iPhone, updates to styles.css

HMalloy committed:
pulled testing data out of README.md and created testing.txt based on instructions for assignment. Committing to show troubleshooting of IE 11 javascript issue.

HMalloy committed:
re-versed IE11 fix

HMalloy committed:
updated testing to include iPad pro (to get in one more device) and updated README.md to include info about google fonts used on the site

HMalloy committed:
updated personas to include more specifics regarding devices

HMalloy committed:
updated README.md formatting

HMalloy committed:
updates to README.md formatting

HMalloy committed:
more updates to README formatting

HMalloy committed:
more formatting updates to README.md
